cmd/grr/workflow.go

```diff
@@ -24,9 +24,9 @@ func getCmd(registry grizzly.Registry) *cli.Command {
 
 func listCmd(registry grizzly.Registry) *cli.Command {
 	cmd := &cli.Command{
-		Use:   "list <jsonnet-file>",
+		Use:   "list [-d <directory> [<jsonnet-file>]",
```

Comment:
Nit: The file vs dir logic might become simpler if the `-d` flag was of type bool instead, so that the commands would always accept a single arg and the flag controls how it is interpreted

Reply:
yeah, I thought about that recently. We can add `-H` or `--hidden` which switches on hidden element parsing in jsonnet, and remove it from the current Jsonnet approach. Switching to bools helps here.